Webpack warnings

Hello,

I'm getting this warning while packing with webpack:
```
WARNING in ./~/superagent-no-cache/index.js
Module not found: Error: Can't resolve 'ie' in '..../node_modules/superagent-no-cache'
 @ ./~/superagent-no-cache/index.js 3:7-20
 @ ./resources/assets/js/app/api.js
 @ ./resources/assets/js/vuex/actions.js
 @ ./resources/assets/js/vuex/store.js
 @ ./resources/assets/js/index.js
```

--- a/index.js
+++ b/index.js
@@ -1,9 +1,4 @@
-var ie
-try {
-  ie = require('ie')
-} catch (e) {
-  ie = require('component-ie')
-}
+var ie = require('component-ie')
 
 function with_query_strings (request) {
   var timestamp = Date.now().toString()

